src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```diff
@@ -17,6 +17,14 @@ $avatarUrl: var(--user-profile-picture-avatar-url, none);
 
   cursor: pointer;
 
+  &__onlyIcon {
+    width: 100px;
+    height: 100px;
+    bottom: 0;
+    left: 0;
+    margin: 7px auto;
```

Comment:
not even divisible by 2 like those `$spacing--` constants

Reply:
replaced with $spacing--sm